### DISCUSSION TITLE
Welcome to janetdocs Discussions!

### DISCUSSION THREAD

**Discussion by swlkr** [swlkr/janetdocs#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by swlkr** [comment]
If you have any cool ideas for janetdocs let me have em!

**Reply by pepe** [reply]
If there is any voting ongoing then #24 is my favourite. But I am afraid it means more manual work, than coding (not that there isn't any).
